mono/mini/interp/interp.c

```diff
@@ -5088,28 +5088,12 @@ interp_exec_method_full (InterpFrame *frame, ThreadContext *context, guint16 *st
 			mono_memory_barrier ();
 			MINT_IN_BREAK;
 		}
-		MINT_IN_CASE(MINT_MONO_JIT_ATTACH) {
+		MINT_IN_CASE(MINT_MONO_THREADS_ATTACH_COOP) {
```

Comment:
AFAIU it doesn't seem we need this opcode: since this opcode is generated when calling `mono_threads_attach_coop ` and all this opcode does is calling `mono_threads_attach_coop `. Why can't we let the icall go through directly through the existing icall calling code?

Reply:
The existing icall calling code - in `do_icall` expects the thread to already be attached (specifically it wants to manipulate the LMF of the current thread).  I tried it as a regular icall and got crashes.  This was simpler.

Reply:
Sounds good 👍